### ISSUE TITLE
Invariant Violation: requireNativeComponent: "ContextMenu" was not found in the UIManager.

### ISSUE THREAD

**Issue by fernando-ropero-akimad** [mpiannucci/react-native-context-menu-view#77]
When using this component I encounter the following error:

<img width="393" alt="Captura de pantalla 2023-03-13 a la(s) 21 16 16" src="https://user-images.githubusercontent.com/122032089/224875485-8e2ff7b4-1f83-4da4-bac7-a153a1f641e7.png">

ios works fine but android gives error.

I've tried:

Restarting computer
Re-adding and re installing with yarn (I don't use npm)
A bunch of stuff with pods I can't even remember

"react-native-context-menu-view": "^1.9.1"
"react-native": "0.69.7",

**Comment by CavidM** [comment]
Same issue with react-native 0.71.8

**Comment by mimo-10** [comment]
same issue now it is 0.78

**Comment by AlanHazel** [comment]
For Expo users: I was able to solve this by simply rebuilding in Expo.

**Comment by williamlmao** [comment]
Same issue - @AlanHazel how did you fix this?

**Comment by 220ish** [comment]
clearing the cache build doesn't help in expo

**Comment by mpiannucci** [comment]
I updated the example app to the latest react native and built fresh and I am not seeing the error unfortunately

**Comment by PMVPYW** [comment]
Can it be the android version?

**Comment by mpiannucci** [comment]
I had this issue at one point on android, and running the following cleared it up: 

```bash
rm -rf node_modules
cd android
./gradlew clean
cd ../
yarn install
react-native run-android --reset-cache
```

**Comment by tiagodocusse** [comment]
Same problem with Expo here. My versions: This lib is 1.14.1, Expo v. 49.0.21, React Native v. 0.72.6.

Tried cleaning Expo cache, expo install --check, start with npx expo start --clear, nothing worked. I get the same error both in Android and iOS.

**Comment by nico-cohesion** [comment]
@mpiannucci please have a look, I am also getting the same issue.

"expo": "~49.0.21"
"react": "18.2.0"
"react-native": "0.72.6"

**Comment by jayan2019** [comment]
Also facing the same issue.
`react-native-context-menu-view: "^1.15.0"`
`expo: "^48.0.0",`
`react-native: "0.71.8",`

**Comment by angelopedroso** [comment]
> For those who are still facing the error on `expo`, you need to -
> 
> 1. Install this library using  `npm install react-native-context-menu-view --save`.
> 2. Run `eas build --profile development` to create a new development build.

I tried it and nothing, I continue getting that error.

**Comment by nico-cohesion** [comment]
Also an issue in:
"expo": "51.0.8"
"react-native": "0.74.1"
"react-native-context-menu-view": "^1.16.0"

**Comment by stefan-stankovic** [comment]
Has anyone fixed this?

**Comment by willashley23** [comment]
Bump. Running into the same issue still. I created a development build, and can get it to run in the simulator using `npx expo run:ios`, but it doesn't work inside Expo Go. Would really appreciate some help here, it looked really nice in the simulator!

**Comment by mpiannucci** [comment]
I don't think it will ever work in expo go. You need to install the dev build on an actual device

**Comment by willashley23** [comment]
Yeah, I realized Expo Go can't render some native components so things like this will only ever work in a dev build.
